pkg/volume/csi/csi_plugin.go

```diff
@@ -96,8 +96,7 @@ var PluginHandler = &RegistrationHandler{}
 // ValidatePlugin is called by kubelet's plugin watcher upon detection
 // of a new registration socket opened by CSI Driver registrar side car.
 func (h *RegistrationHandler) ValidatePlugin(pluginName string, endpoint string, versions []string) error {
-	klog.Infof(log("Trying to validate a new CSI Driver with name: %s endpoint: %s versions: %s",
-		pluginName, endpoint, strings.Join(versions, ",")))
+	klog.InfoS(log("Trying to validate a new CSI Driver"), "name", pluginName, "endpoint", endpoint, "versions", strings.Join(versions, ","))
```

Comment:
do we need log format?

Reply:
I think that's fine to keep the status quo. As structured logging evolves we may discover a more reasonable solution to systematically adding annotations.